pkgs/misc/themes/arc/default.nix

```diff
@@ -1,43 +1,55 @@
-{ stdenv, fetchFromGitHub, autoreconfHook, pkgconfig, gnome3, gtk-engine-murrine }:
+{ stdenv, fetchFromGitHub, sassc, autoreconfHook, pkgconfig, gnome3
+, gnome-themes-standard, gtk-engine-murrine, optipng, inkscape}:
 
 let
-  # treat versions newer than 3.22 as 3.22
-  gnomeVersion = if stdenv.lib.versionOlder "3.22" gnome3.version then "3.22" else gnome3.version;
+  # per README: For GNOME 3.24 and 3.26, use --with-gnome=3.22
+  gnomeVersion = if gnome3.version == "3.24" || gnome3.version == "3.26" then "3.22" else gnome3.version;
   pname = "arc-theme";
 in
 
 stdenv.mkDerivation rec {
   name = "${pname}-${version}";
-  version = "2017-05-12";
+  version = "2018-07-13";
 
   src = fetchFromGitHub {
-    owner  = "horst3180";
+    owner  = "NicoHood";
     repo   = pname;
-    rev    = "8290cb813f157a22e64ae58ac3dfb5983b0416e6";
-    sha256 = "1lxiw5iq9n62xzs0fks572c5vkz202jigndxaankxb44wcgn9zyf";
+    rev    = "be675c9e8c853dd4a20f071b408158b0d871f1a4";
+    sha256 = "1kkfnkiih0i3pds5mgd1v9lrdrp4nh4hk42mw7sa4fpbjff4jh6j";
   };
 
-  nativeBuildInputs = [ autoreconfHook pkgconfig ];
-  buildInputs = [ gnome3.gtk ];
+  preBuild = ''
+    # Shut up inkscape's warnings
+    export HOME="$NIX_BUILD_ROOT"
+  '';
 
-  propagatedUserEnvPkgs = [ gtk-engine-murrine ];
+  nativeBuildInputs = [ autoreconfHook pkgconfig sassc optipng inkscape ];
+  buildInputs = [ gnome3.gtk ];
 
-  preferLocalBuild = true;
+  propagatedUserEnvPkgs = [ gnome-themes-standard gtk-engine-murrine ];
```

Comment:
I just learned about `propagatedUserEnvPkgs` today, but there seems to be a movement to deprecate it: https://github.com/NixOS/nixpkgs/issues/43049. Although currently this is how all packages use `gtk-engine-murrine`, and it seems like it's needed for those, because the result isn't a binary you can just wrap.